servicetalk-http-netty/src/test/java/io/servicetalk/http/netty/SniTest.java

```diff
@@ -175,11 +180,12 @@ void noSniClientDefaultServerFallbackSuccess(SslProvider serverSslProvider, SslP
             "{displayName} [{index}] serverSslProvider={0} clientSslProvider={1} protocol={2} useALPN={3}")
     @MethodSource("params")
     void noSniClientDefaultServerFallbackFailExpected(SslProvider serverSslProvider, SslProvider clientSslProvider,
-                                                      HttpProtocol protocol, boolean useALPN) throws Exception {
+                                                      HttpProtocol protocol, boolean useALPN, boolean acceptInsecure)
```

Comment:
Consider enhancing tests to also execute plaintext path, similar to what you do in `OptionalSslTest`: create a non-secure client and test that it can/cannot serve a request depending on `acceptInsecure` flag. We will need to enhance `newSslVerifyService()` to allow plaintext connections.